contracts/KlerosLiquidProxy.sol

```diff
@@ -0,0 +1,245 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./interfaces/IKlerosLiquidProxy.sol";
+import "./interfaces/IExtendedKlerosArbitrator.sol";
+
+contract KlerosLiquidProxy is IKlerosLiquidProxy {
+    IClaimsManager public immutable override claimsManager;
+
+    IArbitrator public immutable override klerosArbitrator; // Kleros expects this exact name
+
+    bytes public override klerosArbitratorExtraData; // Kleros expects this exact name
+
+    mapping(uint256 => uint256) public override disputeIdToClaimIndex;
+
+    mapping(uint256 => uint256) public override claimIndexToDisputeId;
+
+    modifier onlyClaimForwardedToKleros(uint256 claimIndex) {
+        require(claimIndexToDisputeId[claimIndex] != 0, "Invalid claim index");
+        _;
+    }
+
+    constructor(
+        address _claimsManager,
+        address _klerosArbitrator,
+        bytes memory _klerosArbitratorExtraData,
+        string memory _metaEvidence
+    ) {
+        claimsManager = IClaimsManager(_claimsManager);
+        klerosArbitrator = IArbitrator(_klerosArbitrator);
+        klerosArbitratorExtraData = _klerosArbitratorExtraData;
+        emit MetaEvidence(0, _metaEvidence);
+    }
+
+    function forwardDisputeToKlerosArbitrator(uint256 claimIndex)
+        external
+        payable
+        override
+    {
+        (
+            ,
+            IClaimsManager.ClaimStatus claimStatus,
+            address claimant,
+            ,
+            ,
+            ,
+            string memory evidence
+        ) = claimsManager.claims(claimIndex);
+        require(
+            claimStatus == IClaimsManager.ClaimStatus.DisputeCreated,
+            "Dispute not created"
+        );
+        require(
+            claimIndexToDisputeId[claimIndex] == 0,
+            "Dispute already forwarded"
+        );
+        require(
+            claimsManager.claimIndexToArbitrator(claimIndex) == address(this),
+            "Different arbitrator"
+        );
+        uint256 disputeId = klerosArbitrator.createDispute{value: msg.value}(
+            uint256(type(IClaimsManager.ArbitratorDecision).max),
```

Comment:
Is this the same as saying `IClaimsManager.PaySettlement`? I'm guessing we always want to use the last element of the enum for some reason, right?

Reply:
Here we're telling Kleros how many choices they can make https://github.com/kleros/kleros-interaction/blob/master/contracts/standard/arbitration/Arbitrator.sol#L54